### PR TITLE
Add a little text above map preview explaining how you can manipulate it

### PR DESCRIPTION
**Pull request by TheKeyBlue**
Tokyto told me a lot of people don't know you can use the mappreview to move yourself or close/open slots.
So he asked if we could maybe add some text explaining the feature.
Here is what i got.

naamloos2

### CODE REVIEW

**Comment by aeoncleanse** [comment]
How does it look with centred text?

**Comment by TheKeyBlue** [comment]
Maybe this is better?
naamloos3

**Comment by aeoncleanse** [comment]
I think so. Will get more feedback

**Comment by MrNukealizer** [comment]
Have you considered only showing the open/close slot line to the host?

**Comment by speed2CZ** [comment]
How about showing it as a tooltip when you move cursor over the minimap?

**Comment by TheKeyBlue** [comment]
Okay turned it into a tooltip.
Host and non-hosts have different messages.

File: lua/ui/lobby/lobby.lua [hunk 1]
```diff
@@ -2681,6 +2681,20 @@ function CreateUI(maxPlayers)
...
     LayoutHelpers.DepthOverParent(GUI.mapPanel, GUI.panel, 2)
 
+    -- Map Preview Info Labels
+    local tooltipText = {}
+    tooltipText['text'] = LOC("<LOC lobui_0772>Map Preview")
+    -- Map Preview Info Labels
+    if isHost then
+        tooltipText['body'] = LOC("<LOC lobui_0769>Left click ACU icon to move yourself or swap players.") 
+                                    .. '\n' 
+                                    .. LOC("<LOC lobui_0770>Right click ACU icon to close or open the slot.")
```
Comment (speed2CZ) [inline lua/ui/lobby/lobby.lua]: this way is more elegant:
```LUA
tooltipText['body'] = LOCF("%s\n%s", "<LOC lobui_0769>Left click ACU icon to move yourself or swap players.", "<LOC lobui_0770>Right click ACU icon to close or open the slot.")
```

Comment (TheKeyBlue) [inline lua/ui/lobby/lobby.lua]: okay, changed it

**Pull request merged**